tamr_unify_client/dataset/resource.py

```diff
@@ -41,12 +41,12 @@ def version(self):
     @property
     def tags(self):
         """:type: list[str]"""
-        return self._data.get("tags")
+        return [*self._data.get("tags")]
```

Comment:
nit: Idiomatic way to copy lists in python is `[:]`

```suggestion
        return self._data.get("tags")[:]
```